sdk/core/Azure.Core/samples/Diagnostics.md

```diff
@@ -94,7 +151,7 @@ Azure SDKs are instrumented for distributed tracing using ApplicationsInsights o
 
 Application Insights, a feature of Azure Monitor, is an extensible Application Performance Management (APM) service for developers and DevOps professionals. Use it to monitor your live applications. It will automatically detect performance anomalies, and includes powerful analytics tools to help you diagnose issues and to understand what users actually do with your app
 
-If your application already uses ApplicationInsights, automatic collection of Azure SDK traces is supported since version `2.12.0`. 
+If your application already uses ApplicationInsights, automatic collection of Azure SDK traces is supported since version `2.12.0`.
```

Comment:
> If your application already uses ApplicationInsights, automatic collection of Azure SDK traces is supported since version `2.12.0`.

Should we specify the package name or link to NuGet?